### PR TITLE
Create Avatar by upload .glb file on admin system 

### PR DESCRIPTION
**Pull request by kimenyikevin**
## Summary

_A summary of changes being made in this PR_

## Checklist
- [x] Pre-push checks pass `npm run check`
  - [x] Linter passing via `npm run lint`
  - [x] Unit & Integration tests passing via `npm run test:packages`
  - [x] Docker build process passing via `npm run build-client`
- [x] If this PR is still a WIP, convert to a draft 
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewers

## References

_References to pertaining issue(s)_

## QA Steps

1. `git checkout pr_branch_name`
2. `npm install`
3. `npm run dev-reinit`
4. `npm run dev`

List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos.

## Reviewers

_Reviewers for this PR_

### CODE REVIEW

**Comment by zulqarnainhanif** [comment]
@kimenyikevin `npm run format` in the your repo to fix the linting warnings

File: packages/client-core/src/admin/services/AvatarService.ts [hunk 1]
```diff
@@ -73,9 +74,24 @@ export const AvatarService = {
...
-  createAdminAvatar: async (data: any) => {
+  createAdminAvatar: async (blob: Blob, thumbnail: Blob, data: any) => {
     const dispatch = useDispatch()
     try {
+      const uploadArguments: AdminAssetUploadType = {
+        type: 'admin-file-upload',
+        files: [blob, thumbnail],
+        args: [
+          {
+            key: 'avatars/public/CyberbotGold.glb',
```
Comment (zulqarnainhanif) [inline packages/client-core/src/admin/services/AvatarService.ts:85]: due to this fixed argument any avatar i upload update the CyberGold.glb entry

File: packages/client-core/src/admin/services/AvatarService.ts [hunk 2]
```diff
@@ -73,9 +74,24 @@ export const AvatarService = {
...
+  createAdminAvatar: async (blob: Blob, thumbnail: Blob, data: any) => {
     const dispatch = useDispatch()
     try {
+      if (blob) {
+        const uploadArguments: AdminAssetUploadType = {
+          type: 'admin-file-upload',
+          files: [blob, thumbnail],
+          args: [
+            {
+              key: `avatars/public/${thumbnail['name']}`,
```
Comment (zulqarnainhanif) [inline packages/client-core/src/admin/services/AvatarService.ts:86]: to make it more uniwue you can append unicode timestamp with file name as well

**Comment by zulqarnainhanif** [comment]
@kimenyikevin 
- need to show a centralized face view of the avatar in image
Screenshot 2022-03-01 at 9 09 59 AM

- there should be option to upload file in edit menu as well
Screenshot 2022-03-01 at 9 10 28 AM

- You can centralised or make a full width view of upload button and avatar uploaded preview.
Screenshot 2022-03-01 at 9 13 18 AM

File: packages/client-core/src/admin/services/AvatarService.ts (hunk 2 above)
Comment (kimenyikevin) [reply packages/client-core/src/admin/services/AvatarService.ts:86]: I made changes from backend, That is where i modify key property

**Comment by zulqarnainhanif** [comment]
@kimenyikevin the following things needs to be fixed:

- The view model of avatar is neither showing avatar nor its thumbnail correctly
Screenshot 2022-03-04 at 12 20 04 AM

- After updating an avatar if you click view again it shows the old avatar until a manual refresh is done.

- Avatar edit model isn't showing full width view of avatar like create model
Screenshot 2022-03-04 at 12 17 28 AM

**Review by zulqarnainhanif — APPROVED** [review]

**Pull request merged**